src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -83,18 +83,21 @@ def main(verbose: int, quiet: bool):
 )
 @output_option
 def run(input: str, config: str, rules: str, rejects: str, output: str):
-    # Implemented `meta` param in `lexical_index_to_sssom`
-
-    meta = get_metadata_and_prefix_map(config)
+    sssom_meta = get_metadata_and_prefix_map(config)
+    oak_meta = {'curie_map': sssom_meta[0].prefix_map}
```

Comment:
## Bugfix for `AttributeError: 'tuple' object has no attribute 'pop'` @matentzn @hrshdhgd 
@hrshdhgd I don't think is going to be back for a while so, @matentzn should just review.

<details><summary>Details</summary>
<p>

The error was being thrown in OAK, in `lexical_indexer.py`, in `lexical_index_to_sssom()`. However the problem is that the wrong thing was being passed. `lexical_index_to_sssom()` asks for a `meta` object of type `Dict`, but the `meta` that was being passed was a `tuple[Converter, Dict]`. It seemed that what `lexical_index_to_sssom()` really wanted was a dictionary with a `curie_map` in it. 

## Possible side effects
`meta` is used in 2 places in `lexical_index_to_sssom():
1. `Converter.from_prefix_map((meta or {}).pop(CURIE_MAP, {})),`
2. `msdf = MappingSetDataFrame.from_mappings(mappings=mappings, metadata=meta, converter=converter)`

My fix above fixes the problem for (1). But for (2), I don't yet know how `meta` ends up getting used, so I do not know if it would benefit more from just `curie_map` being in `meta`. I don't know how this was when Harshad originally set it up. And when I interrogate the code, it gets difficult to inspect, so my decision for now is: If (2) executes without error, I'm considering that fine for now. My other failsafe for this is that when this PR is done, I'll provide some outputs and have Nico check them. Hopefully, if there's any negative side effects caused by my necessary changes to `meta`, we will notice at that stage.

</p>
</details>

Reply:
I think it is used - but maybe this is not relevant for the mondo-ingest pipeline. We can use other postprocessing methods to append any metadata if needed. Probably your fix is the right direction.

Reply:
Sounds good. I'll leave this open until we've reviewed the outputs to make sure they look good then I'll mark resolved